61.md

```diff
@@ -0,0 +1,36 @@
+NIP-61
+======
+
+Danmaku
+-------
+
+`draft` `optional`
+
+This NIP defines a new kind of event, `2333`, for danmaku.
+
+Danmaku (弾幕 だんまく; 弹幕; literally translated as "bullet curtain" and figuratively as "barrage") is a subtitle system in online video platforms that originates from Japan and popularised in Mainland China. Such system allows user to post moving comments onto a playing video that are synchronized to the video timeline. The comments are typically presented as “shooting” across the screen, resembling a barrage.
+
+### Format
+
+The format uses a regular event kind `2333`.
+
+The `.content` of these events is the content of the danmaku comment.
+
+The list of tags are as follows:
+* `i` (required) universally unique identifier of the video. The format is `<platform (lowercase)>:<identity>` where `<platform>` is the platform of the video and `<identity>` is the unique identifier of the video on that platform. For example, a YouTube video with the identifier `aA-jiiepOrE` would have the identifier `youtube:aA-jiiepOrE`.
+* `time` (required) the time of the video when the danmaku comment is sent in seconds.
```

Comment:
I would like to see an optional `duration` tag. This would be useful for slightly different use cases from danmaku, for example the little comment/link boxes that used to hover over youtube videos.

Reply:
Great idea. We can also add optional tags like color, speed, and so on.